pimd/pim_nht.c

```diff
@@ -574,20 +575,11 @@ static int pim_ecmp_nexthop_search(struct pim_instance *pim,
 					nexthop->mrib_route_metric =
 						pnc->metric;
 					if (PIM_DEBUG_PIM_NHT) {
```

Comment:
Please remove the `{ }` if they are not needed anymore on these debug logs

Reply:
Done